model.js

```diff
@@ -21,4 +21,14 @@ exports.selectArticleById = (article_id) => {
 exports.selectUsers = () => {
     return db.query("SELECT * FROM users")
     .then(data => data.rows)
+}
+
+exports.updateArticles = (article_id, data) => {
+    return db.query(`SELECT votes FROM articles WHERE article_id=$1`, [article_id])
+    .then(data => {
+        return data.rows[0]
+    }).then(({votes}) => {
+        votes+=data.inc_votes
+        return db.query(`UPDATE articles SET votes=$2 WHERE article_id=$1 RETURNING *`, [article_id, votes])
```

Comment:
This could be simplified into one query rather than selecting, incrementing a variable and then updating. Have a look at the UPDATE notes at the bottom of this notes page https://notes.northcoders.com/courses/js-back-end/sql-introduction